spec/classes/mod/php_spec.rb

```diff
@@ -119,7 +119,7 @@
 
                 it { is_expected.to contain_apache__mod('php') }
                 it { is_expected.to contain_file('php.load').with(content: "LoadModule php_module modules/libphp.so\n") }
```

Comment:
I think this is also needed to please Rubocop
```suggestion
                  it { is_expected.to contain_apache__mod('php') }
                  it { is_expected.to contain_file('php.load').with(content: "LoadModule php_module modules/libphp.so\n") }
```